### DISCUSSION TITLE
Is there a way to add description meta tags to the tags listing pages?

### DISCUSSION THREAD

**Discussion by talonx** [facebook/docusaurus#10421] [Q&A] [answered]
E.g. https://blog.incidenthub.cloud/tags/availability shows all the posts tagged with "availability". However, this page itself does not have any meta tags (individual posts do). Is there any way to do this?

Blog mode is on on the site.

**Comment by slorber** [comment]
What do you mean by "meta tags"?

**Reply by Josh-Cena** [reply]
`<meta name="description">` I suppose.

**Reply by talonx** [reply]
> What do you mean by "meta tags"?

The `<meta name="description">` tag

**Reply by slorber** [reply, marked as answer]
@talonx if you add a `description` to tags in `tags.yml`, that page will have a meta `description` and `og:description`.

See our init template: https://tutorial.docusaurus.io/blog/tags/facebook/

```yml
facebook:
  label: Facebook
  permalink: /facebook
  description: Facebook tag description
```

![CleanShot 2024-08-20 at 11 46 51](https://github.com/user-attachments/assets/08bb5d0b-92d8-452b-b62d-3f07e27647f8)

**Reply by talonx** [reply]
> @talonx if you add a `description` to tags in `tags.yml`, that page will have a meta `description` and `og:description`.
> 
> See our init template: https://tutorial.docusaurus.io/blog/tags/facebook/
> 
> ```yaml
> facebook:
>   label: Facebook
>   permalink: /facebook
>   description: Facebook tag description
> ```
> 
> ![CleanShot 2024-08-20 at 11 46 51](https://private-user-images.githubusercontent.com/749374/359426790-08bb5d0b-92d8-452b-b62d-3f07e27647f8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyMTIwNzIsIm5iZiI6MTcyNDIxMTc3MiwicGF0aCI6Ii83NDkzNzQvMzU5NDI2NzkwLTA4YmI1ZDBiLTkyZDgtNDUyYi1iNjJkLTNmMDdlMjc2NDdmOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyMVQwMzQyNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZjAzNTAyN2FhMzIzMDc1ZDlmOWIxZmQ1MTNlOWFiMmU5OTI2ODcxMDY1NzcyMzc2YWJkZmJhZTQ5ODI4ZjdlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.kz_vnreF9NEUrlcnZNHVyFNU2ItYl8KN4v9dBqUFXsI)

That solved it for me, thanks!
